types/appdynamics/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for appdynamics 21.8
+// Project: https://github.com/npm/npm
```

Comment:
Agree, this should be changed to some kind of project link, even if it's not the project source code.

Reply:
No worries at all - I've updated it to point to https://www.appdynamics.com/supported-technologies/nodejs

Is this the sort of page you're thinking?

The other page that could work in my mind is:
https://docs.appdynamics.com/21.6/en/application-monitoring/install-app-server-agents/node-js-agent

Reply:
Hi @weswigham! Are you able to re-review this PR? It think it's ready now